### PR TITLE
Update team page after Alberto leaved, Giuseppe's joined

### PR DESCRIPTION
**Pull request by giuscri**
https://www.notion.so/banksealer/Update-team-page-on-the-website-e8d853031b7c41638d3cb69186f0e697

### CODE REVIEW

**Comment by giuscri** [comment]
Please @neomusic, push a commit filling [placeholder](https://github.com/buildo/banksealer-website/pull/55/commits/e0cc54b9c98d73266883704ebc45f74a7035f165#diff-2d4809bc8d04cb135105698fcbfe5294R216) with your biography

**Comment by neomusic** [comment]
@giuscri I added my biography. Thanks you for your patience

File: team.html [hunk 1]
```diff
@@ -213,7 +213,12 @@ <h4 class="text-sm-center text-primary py-3 text-center text-capitalize team-nam
             Giuseppe Aremare</h4>
           <p class="text-center team-roles text-uppercase"><span
               style="font-size: 11.2px; font-weight: normal;">Software Engineer</span></p>
-          <p class="my-1 text-white white-balance text-left">...</p>
+          <p class="my-1 text-white white-balance text-left">
+            Giuseppe has always been passionate about technology,
+            graduating as a computer expert at Lonato del Garda's technical institute. <br/>
+            He has worked for five years in a software house, building his knowledge in web development, SEO and digital assistants. <br/>
+            Driven by his passion, he loves to take on new challenges and create innovative and well written software.
+          </p>
```
Comment (giuscri) [inline team.html]: I'd rewrite this as

> Giuseppe has always been passionate about technology. He graduated at Institute of Higher Education Luigi Cerebotani as Expert in Computer Science. He worked five years for a software house, building his knowledge in web development, SEO and digital assistants. Driven by his passion, he loves to take on new challenges and to create innovative and well written software.

File: team.html [hunk 2]
```diff
@@ -219,6 +206,33 @@ <h4 class="text-sm-center text-primary py-3 text-center text-capitalize team-nam
...
+            alt="group icon">
+          <h4 class="text-sm-center text-primary py-3 text-center text-capitalize team-name" contenteditable="true">
+            Giuseppe Aremare</h4>
+          <p class="text-center team-roles text-uppercase"><span
+              style="font-size: 11.2px; font-weight: normal;">Software Engineer</span></p>
+          <p class="my-1 text-white white-balance text-left">
+            Giuseppe has always been passionate about technology, graduating as a computer expert at Lonato del
+            Garda's technical institute. He has worked for five years in a software house, building his knowledge in
+            web development, SEO and digital assistants. Driven by his passion, he loves to take on new challenges
+            and create innovative and well written software.</p>
```
Comment (giuscri) [inline team.html]: I'd rewrite this as

> Giuseppe has always been passionate about technology. He graduated at Institute of Higher Education Luigi Cerebotani as Expert in Computer Science. He worked five years for a software house, building his knowledge in web development, SEO and digital assistants. Driven by his passion, he loves to take on new challenges and to create innovative and well written software.

**Comment by giuscri** [comment]
@calippo would you review this please?

**Comment by giuscri** [comment]
someone has to merge this :)

**Pull request merged**